### PR TITLE
Pro button

### PR DESCRIPTION
**Pull request by justtry**
create applicant page png 2018-06-28 14-33-15
Updates for `/admin/create-applicant` page.

branch | PR
------ | ------
1639-create-applicant-page | [Auto](https://github.com/x-team/auto/pull/1258)

### CODE REVIEW

File: src/components/ui/Button.js [hunk 1]
```diff
@@ -175,6 +177,28 @@ const extraStyles = {
...
     }
+  `),
+
+  pro: cmz(`
+    & {
+      border-top-left-radius: 0;
+    }
+    &:before {
+      transition: all .3s ease-out
+      content: 'pro'
```
Comment (bernardodiasc) [inline src/components/ui/Button.js]: Hey @justtry, as a suggestion, you can use something like `content: attr(data-tag);` and use the `data-tag` attrb in the element. This way is possible to provide the tag value from props :)

Comment (justtry) [reply src/components/ui/Button.js]: cool, thx @bernardodiasc, will do!)

**Comment by nicksp** [comment]
Hey guys, just to clarify things. We aren't working on this PR for now, as this is a different scope and planned to be investigated and developed in a separate ticket. So, this is on hold until we start on that task.

**Comment by bernardodiasc** [comment]
Closing this PR in favor of https://github.com/x-team/auto-ui/pull/256

**Pull request closed without merging**